addons/mail/static/src/js/activity.js

```diff
@@ -406,7 +406,7 @@ const KanbanActivity = AbstractField.extend({
      * @private
      */
     _reload() {
-        this.trigger_up('reload', { db_id: this.record_id, keepChanges: true });
+        this.trigger_up('reload', { db_id: this.record.id, keepChanges: true });
```

Comment:
ok, I'm that that super knowledge of `AbstractField`, but I think it's not possible to have `undefined` for `this.record`.